Source/Samples/Sample/TK.CustomMap.Sample.Droid/Properties/AndroidManifest.xml

```diff
@@ -1,9 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
-	<uses-sdk android:minSdkVersion="15" />
+	<uses-sdk android:minSdkVersion="15" android:targetSdkVersion="27" />
```

Comment:
we should be able to do without the target SDK version

Reply:
say ok! i agree